### ISSUE TITLE
Archive this project

### ISSUE THREAD

**Issue by legalsylvain** [OCA/webhook#11]
Hi.
This project doesnt' contains any module (from version 9.0 to version 16.0)
There are 2 PR never merged against 10.0
proposing to drop it / archive it.

What do you think ? 

@lasley, @sbidoul

**Comment by sbidoul** [comment]
Yes we should archive it. Not sure how our tooling is going to deal with archived projects, tough.

**Comment by legalsylvain** [comment]
> Not sure how our tooling is going to deal with archived projects, tough.

Yes, that was the implied question. First time in the OCA ?

**Comment by legalsylvain** [comment]
Up ! @OCA/board

**Comment by simahawk** [comment]
We have new PRs since then... do you want to re-route them?

**Comment by simahawk** [comment]
> Yes we should archive it. Not sure how our tooling is going to deal with archived projects, tough.

We archive it and remove this https://github.com/OCA/repo-maintainer-conf/blob/master/conf/repo/webhook.yml
